DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/tools/email-template/basic-information-on-email-template.md

MultiSafepay offers a number of options for sending a standard email, for example, a successful payment or a reminder email also known as a Second Chance email and many more. 

> > MultiSafepay is unable to customize payment pages or email templates for you, although if you do encounter functionality issues, contact us at <integration@multisafepay.com>

Reviewer: ```suggestion
> MultiSafepay cannot customize payment pages or email templates for you. For functionality support, email the Integration Team at <integration@multisafepay.com>
```
